components/AddContributorsHeader/add-contributors-header.tsx

```diff
@@ -13,6 +13,7 @@ import useDebounceTerm from "lib/hooks/useDebounceTerm";
 interface AddContributorsHeaderProps {
   selectedContributorsIds: number[];
   list: DbUserList;
+  workspaceId?: string;
```

Comment:
The places where `workspaceId` is optional is temporary until we flip the switch to enable workspaces for everyone?

Reply:
Yep, these are used in both lists and workspaces for now